cmd/insprd/operators/nodes/converter_test.go

```diff
@@ -96,7 +96,7 @@ func Test_intToint32(t *testing.T) {
 	}
 }
 
-func TestNodeOperator_withBoundary(t *testing.T) {
+func TestNodeOperator_withBoundary(t *testing.T) { // needs more testing, longer creates, updates, and deletes
```

Comment:
add a `TODO` then, easier to go around the project searching for a specific tag than to find this comment